lib/commands/os/configure.ts

```diff
@@ -488,3 +488,39 @@ function camelifyConfigOptions(options: FlagsDef): { [key: string]: any } {
 		return key;
 	});
 }
+
+/**
+ * Returns the boot partition number of the given image.
+ * @param imagePath Local filesystem path to a balenaOS image file
+ */
+async function getBootPartition(imagePath: string): Promise<number> {
+	const filedisk = require('file-disk');
+	const partitioninfo = require('partitioninfo');
```

Comment:
I gather that [file-disk](https://github.com/balena-io-modules/file-disk) and [partitioninfo](https://github.com/balena-io-modules/partitioninfo) are both TypeScript projects, so they probably have usable typing information. `require()` clauses discard type information, so the `filedisk` and `partitioninfo` variables would currently have type `any`. Try replacing them with:
```suggestion
	const filedisk = await import('file-disk');
	const partitioninfo = await import('partitioninfo');
```
Hopefully there won't be complicated type mismatch problems to resolve (sometimes it doesn't feel worth it, but many times it also reveals bugs that can be fixed before users come across them).

Reply:
That did the trick, thank you for pointing it out. 😉